### PR TITLE
[MNOE-501] UI improvement on-boarding

### PR DESCRIPTION
**Pull request by rheasunshine**
@ouranos Please review

### CODE REVIEW

File: src/app/views/onboarding/step2.less [hunk 1]
```diff
@@ -1,117 +1,125 @@
 #step2 {
   .apps-container {
+    display: -ms-flex;
+    display: -webkit-flex;
     display: flex;
```
Comment (alexnoox) [inline src/app/views/onboarding/step2.less]: You don't need to add browser specific instructions, we add them at build time using [postcss](https://github.com/postcss/postcss) which is using [autoprefixer](https://github.com/postcss/autoprefixer).

File: src/app/views/onboarding/step2.html [hunk 2]
```diff
@@ -7,34 +7,37 @@ <h1 translate>mno_enterprise.templates.onboarding.select_your_app.title</h1>
...
-              </div>
-            </mno-widget-header>
-            <mno-widget-body class="no-padding large">
-              <div class="app-tile" ng-repeat="app in vm.marketplace.apps | filter:vm.appsFilter | filter:{name: vm.searchTerm}">
-                <a class="app-info-modal" href="" ng-click="vm.openInfoModal(app)"><i class="fa fa-info-circle"></i></a>
-                <div class="app-icon" ng-click="vm.toggleApp(app)">
-                  <div class="checked" ng-show="app.checked">
-                    <i class="fa fa-check fa-5x"></i>
+          <div id="apps">
+            <section>
```
Comment (alexnoox) [inline src/app/views/onboarding/step2.html]: I think you can remove this `section` and merge its LESS instructions (if any) in `.apps`. This is to keep the DOM simple. Usually, you have one container and its elements right under it.
If you have a good reason to use it, keep it of course.

Comment (rheasunshine) [inline src/app/views/onboarding/step2.html]: It was unnecessary 👍

File: src/app/views/onboarding/step2.less (hunk 1 above)
Comment (rheasunshine) [inline src/app/views/onboarding/step2.less]: got it!

**Comment by rheasunshine** [comment]
@alexnoox  changes pushed 👍

**Comment by rheasunshine** [comment]
What is LGTM? Your old comment had it.

**Comment by alexnoox** [comment]
@rheasunshine LGTM: looks good to me (most popular comment on Github 😄)

**Comment by alexnoox** [comment]
@ouranos Please have a look

**Pull request merged**